### DISCUSSION TITLE
Dark theme seems to bother reading disqus comments

### DISCUSSION THREAD

**Discussion by djkeh** [mmistakes/so-simple-theme#354] [Q&A] [answered]
## Summary

I think dark theme makes users have difficulty reading disqus comments on the bottom of the article.

### as-is

![image](https://user-images.githubusercontent.com/4949544/68266862-bb40cc80-0093-11ea-8f45-ecbdbab5ce64.png)

### to-be

This is my quick work, but you'd probably do even better 👍 

![image](https://user-images.githubusercontent.com/4949544/68267228-0e674f00-0095-11ea-8f02-4693e54f2996.png)

## Motivation

I thought the disqus comments are not quite readable under dark theme.

## Drawbacks

I'm looking forward to your opinion @mmistakes

**Comment by mmistakes** [comment]
What’s it look like if you set the Disqus scheme to dark in its settings?

https://help.disqus.com/en/articles/1717201-disqus-appearance-tweaks

**Comment by djkeh** [comment]
It looks great!

![image](https://user-images.githubusercontent.com/4949544/68269915-e0870800-009e-11ea-80b4-367918bbefeb.png)

Maybe auto scheme didn't work for a while. I'll set the option back to `auto`, give it some time and see if it works as expected.

**Comment by djkeh** [comment, marked as answer]
I think I can close this issue now anyway, @mmistakes let me know that this issue has nothing to do with so simple theme itself, it can be controlled via disqus configuration. It is even supposed to work automatically. Thank you again, @mmistakes!
